error-handler.js

```diff
@@ -258,9 +258,17 @@ createHandler = function createHandler(options) {
         }
       },
 
+      shouldContinue = function
```

Comment:
> OK. This changes the previous/existing behavior of the library.

Of course. The whole point of this PR was that the existing behavior was wrong. =)

> 503 Too Busy errors as they relate to express?

I'm not sure if Express has any automatic 503 handling capabilities. I've never generated a 503 on purpose from Express because all my apps live behind reverse proxies that have maintenance status pages.